<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lime Report Demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../limereport/report.qrc">
    <normaloff>:/report/images/logo32</normaloff>:/report/images/logo32</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Reports</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>59</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="demo_r2.qrc">:/images/images/logo1.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="limeReportLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>127</red>
                    <green>170</green>
                    <blue>24</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>189</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>127</red>
                    <green>170</green>
                    <blue>24</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>189</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>190</red>
                    <green>190</green>
                    <blue>190</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>190</red>
                    <green>190</green>
                    <blue>190</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Lime Report </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DEMO</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>4</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDesign_Report"/>
   <addaction name="separator"/>
   <addaction name="actionPrint_Report"/>
   <addaction name="actionExport_to_PDF"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionOne_to_One"/>
   <addaction name="actionFit_Width"/>
   <addaction name="actionFit_Page"/>
   <addaction name="separator"/>
   <addaction name="actionFirst_Page"/>
   <addaction name="actionPrior_Page"/>
   <addaction name="actionNext_Page"/>
   <addaction name="actionLast_Page"/>
  </widget>
  <action name="actionDesign_Report">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/design.png</normaloff>:/images/images/design.png</iconset>
   </property>
   <property name="text">
    <string>Design Report</string>
   </property>
  </action>
  <action name="actionPrint_Report">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/Print.png</normaloff>:/images/images/Print.png</iconset>
   </property>
   <property name="text">
    <string>Print Report</string>
   </property>
  </action>
  <action name="actionExport_to_PDF">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/PDF.png</normaloff>:/images/images/PDF.png</iconset>
   </property>
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/ZoomIn.png</normaloff>:/images/images/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconText">
    <string/>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/ZoomOut.png</normaloff>:/images/images/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionOne_to_One">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/OneToOne.png</normaloff>:/images/images/OneToOne.png</iconset>
   </property>
   <property name="text">
    <string>One to One</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/FitWidth.png</normaloff>:/images/images/FitWidth.png</iconset>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="actionFit_Page">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/FitPage.png</normaloff>:/images/images/FitPage.png</iconset>
   </property>
   <property name="text">
    <string>Fit Page</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/First.png</normaloff>:/images/images/First.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>First Page</string>
   </property>
  </action>
  <action name="actionPrior_Page">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/Prev.png</normaloff>:/images/images/Prev.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Prior Page</string>
   </property>
  </action>
  <action name="actionNext_Page">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/Next.png</normaloff>:/images/images/Next.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Next Page</string>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="icon">
    <iconset resource="demo_r2.qrc">
     <normaloff>:/images/images/Last.png</normaloff>:/images/images/Last.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Last Page</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="demo_r2.qrc"/>
  <include location="../limereport/report.qrc"/>
 </resources>
 <connections/>
</ui>
